python/tvm/relay/backend/interpreter.py

```diff
@@ -264,6 +244,26 @@ def __init__(self, mod, ctx, target):
         self.target = target
         self._intrp = _backend.CreateInterpreter(mod, ctx, target)
 
+    def _arg_to_ast(self, arg):
```

Comment:
```suggestion
    def arg_to_ast(self, arg):
```
ditto below

Reply:
Can do

Reply:
It turns out it can't be a method because another object also uses it, so I am going to make it a function that takes the mod as a parameter instead

Reply:
The `_` is to mark private in Python, its convention don't change name.